src/AutoBalance.cpp

```diff
@@ -459,6 +459,21 @@ class AutoBalance_AllCreatureScript : public AllCreatureScript
 
         CreatureTemplate const *creatureTemplate = creature->GetCreatureTemplate();
 
+		uint32 mapId = creature->GetMapId();
+		if (mapId == 30 || mapId == 628)
+			return;
+		
+        uint32 maxNumberOfPlayers = ((InstanceMap*)sMapMgr->FindMap(creature->GetMapId(), creature->GetInstanceId()))->GetMaxPlayers();
+        //    SOLO  - By MobID
+        if (GetForcedCreatureId(creatureTemplate->Entry) >= 0)
+        {
+            maxNumberOfPlayers = GetForcedCreatureId(creatureTemplate->Entry); // Force maxNumberOfPlayers to be changed to match the Configuration entry.
+			if (maxNumberOfPlayers == 0)
+			{
+				return;
+			}
```

Comment:
will never work, because zero in maxNumberOfPlayers will never get in because of no one put zero to `AutoBalance.DisabledID=""`

Reply:
@Viste you meant that nobody would put `maxNumberOfPlayers=0` in the conf, right?